### ISSUE TITLE
Provide .crx installer file for chrome, edge, and opera.

### ISSUE THREAD

**Issue by Destroyinator69420** [vtoubiana/TrackMeNot#143]
I have been using trackmenot for years, but when it was banned from the chrome web store, I had no way to use it anymore since you do not provide a .crx file. A .crx file is the packaging format for chrome extensions. Luckily, you can install .crx files manually without enabling developer mode. If I could download the .crx file from the github page, then I could drag and drop it into chrome://extensions, edge://extensions or opera://extensions. You should also upload it to the Microsoft web store and addons.opera.com, since they would be happy to host it if it is banned from the chrome web store. I know Adnauseam is also hosted on addons.opera.com, since that is where I get adnauseam from. You should also provide a .xpi file for Firefox and Thunderbird, since Mozilla may ban the extension like they did bypass paywalls clean. The only complaint that I have is that I need to restart it manually when search engines kick trackmenot. It is also important that trackmenot supports the browser's default search engines by default. A complete list of search engines trackmenot should protect against by default by default are Bing, Yahoo, Duckduckgo, Yelp, Neeva, Ecosia, Qwant, Startpage, Wikipedia, Google, Amazon, Youtube, Ebay, Yandex and You. It is important that we protect against privacy search engines too just in case they glow in the dark.

**Comment by Tyrichyrich** [comment]
Though I am not affiliated with this project, you can find the github-chrome link on the main page.

**Comment by mbalc** [comment]
yeah like, a piece of feedback - right now, the page is super confusing in that it doesn't tell anywhere how to install the thing

in FAQ there's
> Click the 'Install TrackMeNot' link
like, where's the damn link?

there's nothing in your GitHub releases tab

I've no clue even how to install it, even though I'm on Firefox, I see no link to that anywhere in any obvious place either
